tests/helpers.cpp

```diff
@@ -0,0 +1,40 @@
+#include <string>
+#include <memory>
+#include <vector>
+#include <iostream>
+#include "helpers.h"
+
+std::string execute(const std::string& cmd)
+{
+    //todo: silence output to terminal
```

Comment:
has this been done yet?

Reply:
No, but i realized later it didn't make much sense. the caller has flexibility now to pipe output or not. Messing with that in the function itself is confusing and unnecessary